pages/video-detail-edit/video-detail-edit.view.scss

```diff
@@ -0,0 +1,189 @@
+.video-edit-page {
+  margin-top: $topbarHeight + 10px;
+  background-color: $background;
+  width: 100%;
+  @media (max-width: 600px) {
+    margin-bottom: 200px;
+  }
+
+  video {
+    width: 100%;
+  }
+
+  // TODO: move upper!
```

Comment:
Don't forget this comment ;)

Reply:
Jesus told me he was to take care of this. @DevStarlight count on me when doing this and adapt other components or views